snappy_wrappers/wrappers/somatic_variants_checking/wrapper.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+"""Wrapper for summarize information of vcf file after
+somatic variant calling step
+"""
+import os
+
+from snakemake.shell import shell
+
+__author__ = "Pham Gia Cuong"
+__email__ = "pham.gia-cuong@bih-charite.de"
+
+base_dir = os.path.dirname(os.path.realpath(__file__))
+shell(
+    r"""
+# -----------------------------------------------------------------------------
+# Redirect stderr to log file by default and enable printing executed commands
+exec 2> >(tee -a "{snakemake.log.log}")
+set -x
+# -----------------------------------------------------------------------------
+
+# Write out information about conda installation
+conda list > {snakemake.log.conda_list}
+conda info > {snakemake.log.conda_info}
+
+python {base_dir}/summarize-vcf.py --rawvcf {snakemake.input.full_vcf} \
+    --filtered-vcf {snakemake.input.passed_vcf} \
+    --exom-bedfile {snakemake.config[step_config][somatic_variant_checking][target_regions]} \
+    --padding {snakemake.config[step_config][somatic_variant_checking][padding]} \
+    --minimal {snakemake.config[step_config][somatic_variant_checking][minimal_support_read]} \
+    --limited {snakemake.config[step_config][somatic_variant_checking][limited_support_read]} \
+    --ignore-regions {snakemake.config[step_config][somatic_variant_checking][ignore_regions]} \
+    --output {snakemake.output.json}
+
+pushd $(dirname {snakemake.output.json})
+md5sum $(basename {snakemake.output.json}) > $(basename {snakemake.output.json_md5})
+    """
+)
+
+# Compute MD5 sums of logs
+shell(
+    r"""
+md5sum {snakemake.log.log} > {snakemake.log.log_md5}
```

Comment:
The md5 checksum file should not contain the path to the file, only the filename.
You can achieve that by:
```
f=$(basename {snakemake.log.log})
d=$(dirname {snakemake.log.log})
pushd $d
md5sum $f > $f.md5
popd
```